numba/tests/npyufunc/test_ufuncbuilding.py

```diff
@@ -317,6 +320,58 @@ def test_guvectorize_invalid_layout(self):
         # (error message depends on Numpy version)
 
 
+class NEP13Array:
+    """https://numpy.org/neps/nep-0013-ufunc-overrides.html"""
+    def __init__(self, array):
+        self.array = array
+
+    def __array__(self):
+        return self.array
+
+    def tolist(self):
+        return self.array.tolist()
+
+    def __array_ufunc__(self, ufunc, method, *args, **kwargs):
+        if method != "__call__":
+            return NotImplemented
+
+        return NEP13Array(ufunc(*[np.asarray(x) for x in args], **kwargs))
```

Comment:
Is this change needed for the PR to work?

Reply:
No, but I thought it would be easier to read if two very similar stub classes were declared next to each other.

Reply:
No;  I felt that it would make it easier to understand the test ccode if these very two similar, and yet subtly different stub classes were presented right next to each other. I can revert if you prefer.